### ISSUE TITLE
feature request: support for recurence

### ISSUE THREAD

**Issue by davidbeijinho** [tiddly-gittly/tiddlywiki-calendar#70]
full calendar supports recurrence with **daysOfWeek** and **startRecur** **endRecur**
https://fullcalendar.io/docs/recurring-events#simple-recurrence

would be possiible to support the simple recurrence?

Maybe in the future:
and even Rrule for more advanced rules
https://fullcalendar.io/docs/rrule-plugin

**Comment by linonetwo** [comment]
How should the underlying tiddler be? Maybe there will be only one tiddler, which creates multiple events.

But currently this layout will only load tiddlers with-in the visible period (i.g. a week).  This is of performance concern. 
So last week's tiddler will not be loaded, if it have recurence events, it won't show-up in this week.

**Comment by davidbeijinho** [comment]
I think only one tiddler its ok.
one tiddler = one event , the only details is that event is a repeat event
for the tiddlers that are shown on the calendar that is controlled by the filter right?

`[tag[todo]has[daysOfWeek]]` this will get all `todos` that have `daysOfWeek` field?

**Comment by linonetwo** [comment]
Yes, maybe use a special tag, and always bring tiddlers with these tag in event calendar. This can work.

**Comment by linonetwo** [comment]
supported RRule in latest version.

**Comment by linonetwo** [comment]
Also there is a visualization for viewing all RRule events.
